modules/client/src/tokenVoting/internal/client/methods.ts

```diff
@@ -738,4 +750,54 @@ export class TokenVotingClientMethods extends ClientCore
     }
     return null;
   }
+
+  /**
+   * Checks if the given token is compatible with the TokenVoting plugin
+   *
+   * @param {string} tokenAddress
+   * @return {*}  {Promise<boolean>}
+   * @memberof TokenVotingClientMethods
+   */
+  public async isTokenGovernanceCompatible(
```

Comment:
I think we should name it `isTokenVotingCompatibleToken` or something more descriptive.

Governance compatibility doesn't make much sense IMO